ballerina/tests/04_client_streaming_client.bal

```diff
@@ -45,3 +45,30 @@ isolated function testClientStreaming() returns Error? {
     anydata response = checkpanic ep->receiveString();
     test:assertEquals(<string> response, "Ack");
 }
+
+@test:Config {enable:true}
+isolated function testClientStreamingSendMessageAfterError() returns Error? {
+    HelloWorld7Client helloWorldEp = check new ("http://localhost:9094");
+    LotsOfGreetingsStreamingClient res = check helloWorldEp->lotsOfGreetings();
+
+    check res->sendError(error AbortedError("Aborted for testing"));
+    Error? err = res->sendString("Hey");
+    if (err is Error) {
```

Comment:
Minor suggestion, we can get rid of this parenthesis to adhere to Ballerina styles.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/333/commits/7e19ebef3ef3bc338efe2267e3cc687cf8fd3ff9